cmd/validator/flags/flags.go

```diff
@@ -377,6 +377,20 @@ var (
 		Usage: "Sets the maximum size for one batch of validator registrations. Use a non-positive value to disable batching.",
 		Value: 0,
 	}
+
+	// SourceDataDirFlag defines a path on disk where source Prysm databases are stored. Used for conversion.
+	SourceDataDirFlag = &cli.StringFlag{
+		Name:     "source-data-dir",
+		Usage:    "Source data directory",
```

Comment:
can you use a more descriptive version for the flag usage? https://docs.prylabs.network/docs/prysm-usage/parameters it shows up on this for users

Reply:
you also need to add it here i believe https://github.com/prysmaticlabs/prysm/blob/develop/cmd/validator/usage.go

Reply:
`SourceDataDirFlag` and `TargetDataDirFlag` do not show on `validator --help`, because they are not registered in `usage.go`.

These options are only used (and advertised) when using `validator db convert-complete-to-minimal`.

==> I moved these options directly in `cmd/validator/db/db.go`.